app/services/usps_daily_test_sender.rb

```diff
@@ -0,0 +1,47 @@
+# Mails a letter to the designated receiver
+class UspsDailyTestSender
+  def run
+    if valid_designated_receiver_pii?
+      UspsConfirmationMaker.new(
+        pii: designated_receiver_pii,
+        issuer: nil, # TODO: login.gov internal issuer?
+        profile: profile,
+        otp: otp_from_date,
+      ).perform
+    else
+      raise 'missing valid designated receiver pii'
+    end
+  rescue => err
+    NewRelic::Agent.notice_error(err)
+  end
+
+  # @return [String] 10-digit OTP from the date
+  # @example
+  #   "JAN20_2020"
+  def otp_from_date(date = Time.zone.today)
+    date.strftime('%b%d_%Y').upcase
+  end
+
+  # @return [Hash]
+  def designated_receiver_pii
+    @designated_receiver_pii ||= JSON.parse(
+      AppConfig.env.gpo_designated_receiver_pii,
```

Comment:
We need to make sure we go over this with privacy / security before we add PII to app config.

Reply:
Good point, thanks

Reply:
Got confirmation in Slack: https://gsa-tts.slack.com/archives/CA7NE63SB/p1609870443330800?thread_ts=1609866962.327800&cid=CA7NE63SB